base/docs/basedocs.jl

```diff
@@ -1261,4 +1261,126 @@ available in the `.error` field.
 """
 InitError
 
+"""
+    Any::DataType
+
+`Any` is the union of all types. It has the defining property `isa(x, Any) == true` for any `x`. `Any` describes
```

Comment:
Perhaps `describes therefore` -> `therefore describes`? Or `` `Any` includes the entire...``